### PR TITLE
1611111 - Restore the order of extra arguments in Swift code

### PR DESCRIPTION
**Pull request by badboy**
This fixes 2 small bugs (one is a drive-by fix).

Caution: mozilla/glean will still not compile with this, due to the reason code changes, that don't exist yet for Swift.

### CODE REVIEW

File: glean_parser/swift.py [hunk 1]
```diff
@@ -121,16 +121,18 @@ def output_swift(objs, output_dir, options={}):
...
         "category",
-        "disabled",
-        "lifetime",
         "name",
-        "reason_codes",
         "send_in_pings",
+        "lifetime",
+        "disabled",
         "time_unit",
+        "reason_codes",
```
Comment (mdboom) [inline glean_parser/swift.py]: Why don't we just take this out here.  We can put it back in right before https://github.com/mozilla/glean/pull/649 is ready to land.

File: glean_parser/templates/swift.jinja2 [hunk 2]
```diff
@@ -15,7 +15,7 @@ Jinja2 template is not. Please file bugs! #}
         )
 {% endmacro %}
 
-{% if glean_namespace|length and glean_namespace != "Glean" %}
+{% if glean_namespace|length and glean_namespace is not equalto "Glean" %}
```
Comment (mdboom) [inline glean_parser/templates/swift.jinja2]: Jinja is weird.

Comment (badboy) [reply glean_parser/templates/swift.jinja2]: It is! In all my _local_ and manual tests it worked just fine with `!=`, but not in Glean aparently ¯\_(ツ)_/¯

File: glean_parser/swift.py (hunk 1 above)
Comment (badboy) [reply glean_parser/swift.py]: Need to test it, that might allow us to land the Swift part of reason codes separately.

Comment (badboy) [reply glean_parser/swift.py]: The real problem with reason codes is currently what code is generated:

```swift
let customPing = Ping<NoExtraKeys>(...)
```

We might just want to skip that `<NoExtraKeys>` part for Swift right now or we need to make glean#649 include the Swift part.

Comment (mdboom) [reply glean_parser/swift.py]: Yeah -- I'm working on the Swift part for #649 -- the problem is just what we should land in glean_parser now to make sure glean master is working.

Comment (badboy) [reply glean_parser/swift.py]: Glean master is still on an old parser version, which works. It currently won't work with 1.16.0

Comment (mdboom) [reply glean_parser/swift.py]: Oh -- my bad.  I thought we needed to upgrade glean_parser in glean for other reasons.  If not, let's not worry -- we can but a new glean_parser release and update to it as part of #649.

Comment (badboy) [reply glean_parser/swift.py]: 👍 Sounds good to me.

**Pull request merged**